modules/test/playwright/tests/account-admin-web/accountRole.spec.ts

```diff
@@ -0,0 +1,241 @@
+import getWidgetDefinition from '../layout-content-page-editor-web/utils/getWidgetDefinition';
+
+export const test = mergeTests(
+	accountsPagesTest,
+	applicationsMenuPageTest,
+	commercePagesTest,
+	dataApiHelpersTest,
+	featureFlagsTest({
+		'LPS-178052': true,
+	}),
+	loginTest()
+);
+
+test('LPD-28220 Can user with account manager role view and manage channel defaults', async ({
+	accountManagementWidgetPage,
+	accountsPage,
+	apiHelpers,
+	page,
+}) => {
+	const userAccount = await apiHelpers.headlessAdminUser.postUserAccount();
+
+	userData[userAccount.alternateName] = {
+		name: userAccount.givenName,
+		password: 'test',
+		surname: userAccount.familyName,
+	};
+
+	const site = await apiHelpers.headlessSite.createSite({
+		name: 'Site' + getRandomInt(),
+	});
+
+	apiHelpers.data.push({id: site.id, type: 'site'});
+
+	const layout = await apiHelpers.headlessDelivery.createSitePage({
+		pageDefinition: getPageDefinition([
+			getWidgetDefinition({
+				id: getRandomString(),
+				widgetName:
+					'com_liferay_account_admin_web_internal_portlet_AccountEntriesManagementPortlet',
+			}),
+		]),
+		siteId: site.id,
+		title: getRandomString(),
+	});
+
+	try {
+		const organization =
+			await apiHelpers.headlessAdminUser.postOrganization({
+				name: 'Organization' + getRandomInt(),
+			});
+
+		const accounts = [];
+
+		const account1 = await apiHelpers.headlessAdminUser.postAccount({
+			name: getRandomString(),
+			type: 'business',
+		});
+
+		accounts.push(account1);
+
+		apiHelpers.data.push({id: account1.id, type: 'account'});
+
+		const account2 = await apiHelpers.headlessAdminUser.postAccount({
+			name: getRandomString(),
+			type: 'business',
+		});
+
+		accounts.push(account2);
+
+		apiHelpers.data.push({id: account2.id, type: 'account'});
+
+		await apiHelpers.headlessAdminUser.assignAccountToOrganization(
+			account1.id,
+			organization.id
+		);
+
+		await apiHelpers.headlessAdminUser.assignAccountToOrganization(
+			account2.id,
+			organization.id
+		);
+
+		await apiHelpers.headlessAdminUser.assignUserToAccountByEmailAddress(
+			account1.id,
+			[userAccount.emailAddress]
+		);
+
+		await apiHelpers.headlessAdminUser.assignUserToOrganizationByEmailAddress(
+			organization.id,
+			userAccount.emailAddress
+		);
+
+		await apiHelpers.jsonWebServicesUser.assignUsersToSite(
+			site.id,
+			userAccount.id
+		);
+
+		await performLogout(page);
+		await performLogin(page, userAccount.alternateName);
+
+		await page.goto(`/web/${site.name}/${layout.friendlyUrlPath}`);
+
+		const account1Cell = page.getByRole('cell', {
+			exact: true,
+			name: account1.name,
+		});
+
+		await expect(account1Cell).toBeVisible();
+		const tagName = await account1Cell.evaluate(
+			(element) => element.tagName
+		);
+		expect(tagName).not.toBe('a');
+
+		await expect(
+			page
+				.getByRole('cell', {exact: true, name: account2.name})
+				.getByText(account2.name)
+		).not.toBeVisible();
```

Comment:
this is a locator, it should be in the page... and ... why do you need to do also the getByText? it's not already found by the getByRole?
And I prefer the toHaveCount(0) if possible